Library/Homebrew/formula.rb

```diff
@@ -1008,7 +1008,7 @@ def caveats
   # rarely, you don't want your library symlinked into the main prefix
   # see gettext.rb for an example
   def keg_only?
-    keg_only_reason && keg_only_reason.valid?
+    !!keg_only_reason && keg_only_reason.valid?
```

Comment:
Right, but I think that was a bug. If I add the following line to a formula
```
  keg_only
```
with no reason supplied, presumably I intend `keg_only` to be `true` not `false`.